plugins/modules/aci_config_rollback.py

```diff
@@ -284,7 +284,9 @@ def main():
 
     elif state == "preview":
         aci.path = "mqapi2/snapshots.diff.xml"
-        preview_params = {k: v for d in [{"path": aci.path}, module.params] for k, v in d.items()}
+        preview_path = {"path": aci.path}
+        preview_params = dict(preview_path.items())
+        preview_params.update(module.params.items())
```

Comment:
The assignment to preview path seems redundant to me.
```suggestion
        preview_params =  {"path": aci.path}
        preview_params.update(module.params.items())
```

I don't know if aci.path is used anywhere else besides from these preview params but else the aci.path could be assigned directly in the dict also.

```suggestion
        preview_params =  {"path": "mqapi2/snapshots.diff.xml"}
        preview_params.update(module.params.items())
```

Reply:
1. aci.path is required after the module has run its course through the connection plugin (see: https://github.com/CiscoDevNet/ansible-aci/blob/be1da8186d22c5b4402193a13fc5b928f09b48b1/plugins/module_utils/aci.py#L1536)
2. I made the other modification and looks like it passed the sanity check for Python2.6. So we're good!